doc/api/net.md

```diff
@@ -858,7 +858,7 @@ changes:
     - v17.7.0
     - v16.15.0
     pr-url: https://github.com/nodejs/node/pull/41310
-    description: The `noDelay`, `keepAlive` and `keepAliveInitialDelay`
+    description: The `noDelay`, `keepAlive` and `keepAliveInitialDelay`
```

Comment:
What’s the change here?

Reply:
[Matteo Collina](https://github.com/mcollina)   I looked for a long time and could not figure it out 😁